frontend/layouts/CoreLayout/CoreLayout.jsx

```diff
@@ -21,7 +21,7 @@ const expirationMessage = (
     Your license for Fleet Premium is about to expire. If you’d like to renew or
     have questions about downgrading,{" "}
     <a
-      href="https://github.com/fleetdm/fleet/blob/main/docs/01-Using-Fleet/10-Teams.md#expired_license"
+      href="https://fleetdm.com/docs/using-fleet/faq#how-do-i-downgrade-from-fleet-premium-to-fleet-free"
```

Comment:
This URL is opening the FAQ page, but there doesn't appear to be a Q/A for the anchor we're trying to link to: `#how-do-i-downgrade-from-fleet-premium-to-fleet-free`. @xpkoala Would you please confirm the URL this should be linking to?

Reply:
I think this is an issue with the website itself. The anchor should take the user do that specific question, but doesn't.

Reply:
I don't think that question exists on this page. When I search the page for "downgrade" nothing shows up. When I try to link to another one of the questions that does exist it appears to work as expected: https://fleetdm.com/docs/using-fleet/faq#what-should-i-do-if-my-computer-is-showing-up-as-an-offline-host. So I _think_ the problem is that questions doesn't exist on the FAQ?

Reply:
Ah I see. My mistake.

Reply:
Looking at the associated ticket and Noah updated with the following: 

> Note: Currently, the above FAQ entry does not exist. The following issue covers adding this FAQ entry: #2210

So looks like we're good to merge this as-is, although we may not cherry-pick it into 4.9.0.